python/python/glide/async_commands/core.py

```diff
@@ -2917,6 +2917,42 @@ async def xgroup_del_consumer(
             ),
         )
 
+    async def xgroup_set_id(
+        self,
+        key: str,
+        group_name: str,
+        stream_id: str,
+        entries_read_id: Optional[str] = None,
+    ) -> TOK:
+        """
+        Set the last delivered ID for a consumer group.
+
+        See https://valkey.io/commands/xgroup-setid for more details.
+
+        Args:
+            key (str): The key of the stream.
+            group_name (str): The consumer group name.
+            stream_id (str): The stream entry ID that should be set as the last delivered ID for the consumer group.
+            entries_read_id (Optional[str]): An arbitrary ID (that isn't the first ID, last ID, or the zero ID ("0-0"))
+                used to find out how many entries are between the arbitrary ID (excluding it) and the stream's last
+                entry. This argument can only be specified if you are using Redis version 7.0.0 or above.
```

Comment:
As more I read this, as less I understand it.
No complains to you, you just copied the doc. Do you understand how it changes the command behavior?

Reply:
Yeah to be honest I find the documentation of this argument pretty unclear, I'm not sure how exactly its supposed to effect command output

Reply:
I think this is relevant to the XINFO GROUPS command. If I understand correctly, you can pass this to tell Redis what the last entry received was for the consumer group. Then when you execute XINFO GROUPS it can return this info to you, as well as a count of how many messages have arrived since the indicated entries_read_id